### PR TITLE
Improve cookie modal 1

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently, the cookie modal is not following GDPR standards.

**What is the new behavior (if this is a feature change)?**
- The cookie modal has been improved to follow GDPR standards.

### CODE REVIEW

**Review by Aichnerc — COMMENTED** [review]
Looks okay.

File: src/components/molecules/CookieModal/index.jsx [hunk 1]
```diff
@@ -38,18 +69,63 @@ class ModalPage extends React.Component {
...
-              Wir verwenden Cookies, um die Funktionsfähigkeit der Website
-              aufrecht zu erhalten.
-            </p>
-            <MDBBtn color="agency-red" onClick={this.toggle(1)} size="lg">
-              Ok, danke
-            </MDBBtn>
+            <MDBCol md="6">
+              <MDBBtn color="success" onClick={() => this.checkAll()}>
+                <MDBIcon icon="check-circle" />
+                Alle akzeptieren
```
Comment (schettn) [inline src/components/molecules/CookieModal/index.jsx:121]: I would recommend writing this phrase in the title case.
```suggestion
                Alle Akzeptieren
```

**Review by schettn — CHANGES_REQUESTED** [review]
Please check the case sensitivity. The German language is garbage.

**Review by sweggerat — APPROVED** [review]
looks good

File: src/components/molecules/CookieModal/index.jsx (hunk 1 above)
Comment (Aichnerc) [reply src/components/molecules/CookieModal/index.jsx:121]: It does not matter, once the button text is all uppercase. I could even write `aLlE aKzEpTiErEn`. Has been changed though.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

File: src/components/molecules/CookieModal/index.jsx (hunk 1 above)
Comment (schettn) [reply src/components/molecules/CookieModal/index.jsx:121]: yOu ShOuLd StILl ChOoSe A cOnSiStEnT sHaPe.

**Review by schettn — APPROVED** [review]
LGTM! 😮

**Pull request merged**